### PR TITLE
Added Integration test for Salesforce Connector

### PR DESCRIPTION
**Pull request by hariss63**

### CODE REVIEW

**Comment by RKathees** [comment]
Include in one commit.

File: pom.xml [hunk 1]
```diff
@@ -294,7 +297,7 @@
             <url>http://maven.wso2.org/nexus/content/repositories/snapshots/</url>
             <snapshots>
                 <enabled>true</enabled>
-                <updatePolicy>daily</updatePolicy>
```
Comment (malakasilva) [inline pom.xml]: This needs to be updated daily

Comment (hariss63) [inline pom.xml]: Fixed in https://github.com/wso2-extensions/esb-connector-salesforce/pull/4/

File: src/test/java/org/wso2/carbon/connector/integration/test/salesforce/SalesforceConnectorIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+    @BeforeClass(alwaysRun = true)
+    public void setEnvironment() throws Exception {
+        init("salesforce-connector-1.0.1-SNAPSHOT");
+        apiEndPoint = connectorProperties.getProperty("loginUrl");
+        nameSpaceMap.put("ns", "urn:partner.soap.sforce.com");
+        nameSpaceMap.put("ns1", "urn:sobject.partner.soap.sforce.com");
+    }
+
+    /**
+     * d
```
Comment (hmrajas) [inline src/test/java/org/wso2/carbon/connector/integration/test/salesforce/SalesforceConnectorIntegrationTest.java]: Why this "d" in each method level comments?

Comment (hariss63) [inline src/test/java/org/wso2/carbon/connector/integration/test/salesforce/SalesforceConnectorIntegrationTest.java]: Fixed in https://github.com/wso2-extensions/esb-connector-salesforce/pull/8

**Pull request merged**